lib/src/meteor_client.dart

```diff
@@ -91,13 +91,13 @@ class MeteorClient {
       {};
   final Map<String, Stream<Map<String, dynamic>>> _collectionsStreams = {};
 
-  MeteorClient.connect({required String url, bool debug = false}) {
+  MeteorClient.connect({required String url, bool debug = false, userAgent = 'Dart Meteor / 2.0.3'}) {
```

Comment:
I had a look at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent and found that most examples have no space between `/` and the product name has no space in the between. Should we use `DartMeteor/2.0.3`?